policyengine_uk/parameters/gov/hmrc/tax_free_childcare/investment_income_types.yaml

```
@@ -0,0 +1,14 @@
+description: HMRC excludes certain investment income types when determining eligibility for Tax-Free Childcare, including pension, savings, dividend, and property income
```

Avoid including the parameter values in the description.
```suggestion
description: HMRC excludes these investment income sources when determining eligibility for Tax-Free Childcare.
```